wordcounter/main.go

```diff
@@ -0,0 +1,66 @@
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"sync"
+)
+
+func main() {
+	// For who does not know :D
+	// PLEASE RUN THIS IN TERMINAL: go run *.go data/story1.txt data/story2.txt
+	if len(os.Args) < 2 {
+		fmt.Println("Expected at least one filename.")
+		os.Exit(1)
+	}
+
+	var words []Word
+
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Println(`input words you want to count. example: "a b ab c"`)
+	fmt.Print("> ")
+	for scanner.Scan() {
+		text := scanner.Text()
+		if len(text) == 0 || len(strings.Fields(text)) == 0 {
+			fmt.Println("Expected at least one character.")
+			os.Exit(1)
+		}
+		words = ConvertStringToArrayTypeWord(text)
```

Comment:
In Golang, method with upper case in the first letter is public method implicitly, so we should only public any method if we want reuse by other packages.